mindmeld/converter/dialogflow.py

```diff
@@ -259,13 +260,20 @@ def _create_intent_file(
                 sentence += part
             all_text.append(sentence)
 
-        train, test = train_test_split(all_text, test_size=0.2)
+        for key in default_intent_to_training_file:
+            if key in mindmeld_intent_directory:
+                with open(os.path.join(package_dir, default_intent_to_training_file[key])) as fp:
+                    for line in fp:
+                        all_text.append(line.strip())
```

Comment:
Why do we combine welcome and unrelated data?

Reply:
We are not combining the data from the two intents.